### PR TITLE
feat: Use delegate accounts for price-feeder tx

### PR DESCRIPTION
**Pull request by zarazan**
- Added a delegate-feed-consent CLI call for each price-feeder validator (fixes acoount sequence mismatch error)
- Added option to disable the price-feeder when calling setup-chain. The PF needs to be turned off when running the oracle tests and probably some others or else you will get duplicate pre-votes/votes. All 3 price-feeders need to be running for leverage tests so the only other option would be to add more nodes.
- Added github workflow for the oracle module tests

### CODE REVIEW

File: scripts/chain/helpers/services.sh [hunk 1]
```diff
@@ -8,45 +8,47 @@ command_exists () {
   type "$1" &> /dev/null ;
 }
 
-service_exists() {
+service_running() {
```
Comment (RafilxTenfen) [inline scripts/chain/helpers/services.sh:11]: :100:

**Review by RafilxTenfen — APPROVED** [review]
utACK
